packages/discovery-provider/plugins/pedalboard/apps/backfill-audio-analyses/src/config.ts

```diff
@@ -0,0 +1,45 @@
+import dotenv from 'dotenv'
+import { logger } from './logger'
+import { cleanEnv, str } from 'envalid'
+
+export type Config = {
+  environment: string
+  url: string
+  delegateOwnerWallet: string
+  delegatePrivateKey: string
+  redisUrl: string
+  testRun: boolean
+}
+
+// reads .env file based on environment
+const readDotEnv = () => {
+  const environment = process.env.audius_discprov_env || 'dev'
+  const dotenvConfig = (filename: string) =>
+    dotenv.config({ path: `${filename}.env` })
+  logger.info(`running on ${environment} network`)
+  dotenvConfig(environment)
+}
+
+export const readConfig = (): Config => {
+  readDotEnv()
+
+  // validate env
+  const env = cleanEnv(process.env, {
+    audius_discprov_env: str({ default: 'dev' }),
+    audius_discprov_url: str({ default: '' }),
```

Comment:
this can reliably default to `http://audius-protocol-discovery-provider-1:5000` right?

Reply:
depends on the env; didnt want to bother with any conditional defaults and it should always be present as an envvar anyway